src/pages/not-found/not-found.module.css

```
@@ -0,0 +1,10 @@
+.imageWrapper {
+  width: 70%;
+  margin: 0 auto;
+}
+.image {
+  width: 100%;
+}
+.center {
+  text-align: center;
+}
```

Блоки кода лучше отделять пустой строкой
```suggestion
}

.image {
  width: 100%;
}

.center {
  text-align: center;
}
```